VueJS/src/components/List.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+  <ul>
+    <li v-for="item in items" v-bind:key="item.id">
+      <list-item 
+      v-bind:item="item" 
+      v-on="$listeners" /> <!-- https://stackoverflow.com/a/56172089 -->
+    </li>
+  </ul>
+</template>
+
+<script>
+import ListItem from "./ListItem";
+
+export default {
+  name: "list",
+  components: {
+    "list-item": ListItem
```

Comment:
```suggestion
    ListItem
```
Slightly redundant, since you give the `ListItem` a `name: 'list-item'`. Since `'list-item'` is the kebab cased vesion of `ListItem` which happens to be the default in Vue, you can even omit the `name` in `ListItem.vue`.